Ollivanders/src/net/pottercraft/ollivanders2/spell/RemovePotionEffectInRadius.java

```diff
@@ -57,10 +57,8 @@ public void checkEffect()
         {
             removePotionEffects(player);
         }
-        else
```

Comment:
Why did we remove the else?

Reply:
we want it to do both - affect the person and also affect the radius, not one or the other

Reply:
when affect self is toggled - some we only want to affect everyone else, this is why the caster is handled specifically

Reply:
There are no cases where we only want it to affect the player?

Reply:
You mean only the player - not yet though if there is I will add an affectOnlyPlayer and then put a conditional around this.